grouper/graph.py

```diff
@@ -123,6 +123,9 @@ def __init__(self):
         # Map of groups to their permission grants.
         self.permission_grants = {}  # type: Dict[str, List[GroupPermissionGrant]]
```

Comment:
Nit: renaming to something like `grants_by_group` would help avoid confusion with `_grants_by_permission`. Also I think the class comment is incorrect ("`permission_grants: Permission grant information for users`")

Reply:
I renamed this to `_group_grants` (to parallel `_service_account_grants`) and made it private and fixed the tests that were using it.